translator/translate/otel/processor/awsentity/translator_test.go

```diff
@@ -39,6 +40,74 @@ func TestTranslate(t *testing.T) {
 				Platform:       config.ModeEC2,
 			},
 		},
+		"KubernetesUnderLogs": {
```

Comment:
nit: Would prefer for most of these tests to be in `common_test.go` since that's where the `GetClusterName` function is, but I'm fine with it being here too.